packages/v4/src/content/design-guidelines/usage-and-behavior/lists-and-tables/lists-and-tables.md

```diff
@@ -145,12 +151,30 @@ If a table contains these two attributes: (System Name | Last Sync) you may want
 
 If a table contains these three attributes: (System Name | Last Sync | Severity) you may want to show the system with the highest Severity because that is the system the user should tend to first.
 
+## When to use compact vs. basic spacing
```

Comment:
This section is great! Would maybe add a little intro sentence here saying something like:
"Whether to use a compact table/data list or basic table/data list is up to you and your use case. However, here is some guidance for when to use which:"

Reply:
I also wonder if this should be compact vs. large spacing? I've been using the word "large" where you are using the word "basic" to describe the larger spacing variation

Reply:
The "basic" word comes from the [implementation page](https://www.patternfly.org/v4/documentation/react/components/table)
But, I don't have a problem to rename it. My only concern is that word "larger" isn't that descriptive.